source/expressions.tex

```diff
@@ -1152,13 +1152,16 @@
 \rSec2[expr.prim.literal]{Literals}
 
 \pnum
-A
 \indextext{literal}%
 \indextext{constant}%
-\grammarterm{literal}
-is a primary expression.
-Its type depends on its form\iref{lex.literal}.
-A string literal is an lvalue; all other literals are prvalues.
+A \grammarterm{literal} is a primary expression.
+A \grammarterm{user-defined-literal} is interpreted as a function call
+as described in \ref{lex.ext};
+the type and value category of such a literal is determined by the
+rules for function calls\iref{expr.call}.
+For other literals,
+the type is determined by its form as specified in \ref{lex.literal};
+a \grammarterm{string-literal} is an lvalue; all other literals are prvalues.
```

Comment:
This is a bit too semicolon-heavy for my tastes. How about something like:

> A *literal* is a primary expression, whose type is determined based on its form as specified in [lex.literal]. A *string-literal* is an lvalue, a *user-defined-literal* has the same value category as the corresponding operator call expression described in [lex.ext], and any other *literal* is a prvalue.

Reply:
Fixed.